config/etfuturum/blocksitems.cfg

```diff
@@ -0,0 +1,445 @@
+
+    # Enables stripped log blocks [default: true]
+    B:enableStrippedLogs=true
+}
+
+
+##########################################################################################################
+# misc items
+#--------------------------------------------------------------------------------------------------------#
+# Items that don't fit in any other category.
+##########################################################################################################
+
+"misc items" {
+    #  [default: true]
+    B:enableBeetroot=true
+
+    #  [default: true]
+    B:enableIronNugget=true
+
+    #  [default: true]
+    B:enableLingeringPotions=true
+
+    #  [default: true]
+    B:enableMutton=true
+
+    #  [default: true]
+    B:enableNewDyes=true
+
+    # Appears in stronghold corridor and dungeon chests. [default: true]
+    B:enableOtherside=true
+
+    # Appears in Nether fortress chest loot. [default: true]
+    B:enablePigstep=true
+
+    # If true, vanilla and Et Futurum copper ores will drop raw ore items. [default: true]
+    B:enableRawOres=false
+
+    #  [default: true]
+    B:enableSuspiciousStew=true
+
+    #  [default: true]
+    B:enableTippedArrows=true
+
+    #  [default: true]
+    B:enableTotemUndying=true
+}
+
+
+##########################################################################################################
+# natural blocks
+#--------------------------------------------------------------------------------------------------------#
+# Blocks that can generate naturally in your world. Check world.cfg for generation values.
+##########################################################################################################
+
+"natural blocks" {
+    # Set how hard deepslate is, in case you like the look of deepslate but don't want to deal with the additional hardness. Regular stone is 1.5. [range: 0.0 ~ 100.0, default: 3.0]
+    S:deepslateHardness=2.3333333333
+
+    # Enables tinted glass, amethyst blocks, budding amethyst and amethyst crystals. Also enables the item too. [default: true]
+    B:enableAmethyst=false
+
+    # Enables bamboo wood and all of its wood subtypes, including the bamboo stalks themselves. [default: true]
+    B:enableBambooBlocks=true
+
+    # This must be on for the basalt deltas biome to generate unless Netherlicious is installed. [default: true]
+    B:enableBasalt=true
+
+    #  [default: true]
+    B:enableBlueIce=true
+
+    #  [default: true]
+    B:enableBoneBlock=true
+
+    #  [default: true]
+    B:enableCalcite=true
+
+    # Enables cherry wood and all of its wood subtypes. [default: true]
+    B:enableCherryBlocks=true
+
+    # Enables chorus plants and purpur blocks [default: true]
+    B:enableChorusBlocks=true
+
+    #  [default: true]
+    B:enableCoarseDirt=true
+
+    # Copper ore and copper blocks, variants, and waxed variants. (Slime balls are used if no mod introduces wax and if honey is disabled) [default: true]
+    B:enableCopper=true
```

Comment:
should be false, we dont use vanilla ore gen

Reply:
Done.

Reply:
sorry to come back to this but probably with the other configs copper ore is disabled anyway right? its a bit confusing that there are 3 overlapping copper configs.
so then this would only be the aged and waxed blocks?

Reply:
I believe the comment is perfectly clear. It enables all copper present in the mod as it says. If you are looking for just the decorative stuff, then the option in modcompat is what you need. Although I see you already found it.